docs/examples/agrivoltaics/plot_diffuse_PAR_Spitters_relationship.py

```diff
@@ -0,0 +1,145 @@
+"""
+Calculating the diffuse PAR using Spitter's relationship
+=========================================================
+
+This example demonstrates how to calculate the diffuse photosynthetically
+active radiation (PAR) from total PAR using Spitter's relationship.
+"""
+
+# %%
+# The photosynthetically active radiation (PAR) is a key component in the
+# photosynthesis process of plants. As in photovoltaic systems, PAR can be
+# divided into direct and diffuse components. The diffuse fraction of PAR
+# with respect to the total PAR is important in agrivoltaic systems, where
+# crops are grown under solar panels. The diffuse fraction of PAR can be
+# calculated using the Spitter's relationship [1]_ implemented in
+# :py:func:`~pvlib.irradiance.spitters_relationship`.
+# This model requires the solar zenith angle and the fraction of the global
+# radiation that is diffuse as inputs.
+#
+# .. note::
+#    Understanding the distinction between the global radiation and the PAR is
+#    a key concept. The global radiation is the total amount of solar radiation
```

Comment:
Is global radiation limited to any particular bandwidth in [1]? If it is not, then I'd describe it as "broadband solar radiation" and not mention PV at all. Mentioning PV may imply to some that the global solar radiation means the radiation in the photovoltaic band, usually 400-1100nm.

Reply:
I only see:

> While for PV systems the total global radiation is the key term, for crops, the 38 analogous term is the photosynthetically active radiation (PAR).

but no mention of the bandwidth.